testsuite/features/reposync/srv_check_installer_updates.feature

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2020 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Check Installer Update repositories
```

Comment:
The feature is a feature of SUSE manager.

Change to `Installer update repositories` please.